rpm/fido/Makefile

```diff
@@ -0,0 +1,43 @@
+NAME          = fido
+VERSION       = 1.3.5
+RELEASE	      = 88
```

Comment:
Here you're missing tabs and whitespaces. This is happening consistently across many of the files you're changing in this PR. Can you fix it?